chipsec/logger.py

```diff
@@ -125,31 +125,76 @@ def getMessage(self) -> str:
 class Logger:
     """Class for logging to console, text file, XML."""
 
-    def __init__(self):
+    class level(Enum):
+        DEBUG = 10
+        VERBOSE = 11
+        # HAL = 12
```

Comment:
Why are we removing hal?

Reply:
Not removing it, it was a placeholder. It's now been added.